### DISCUSSION TITLE
Purpose of 'Race'.

### DISCUSSION THREAD

**Discussion by MasterTrainerPK** [FakeFishGames/Barotrauma#7168] [Code questions and modding support]
I was trying to create a source mod to add more humans to the game, allowing players of my mod a few more options, and found that **the enum 'Race', previously used to determine which sprite to use, is still used in the code, despite the reformatted sprites. What is the purpose of of this data, now that it doesn't determine the sprites?**

**Comment by Regalis11** [comment]
It _can_ still be used the same way as before, the vanilla humans just no longer use it.
